### PR TITLE
Add service_mongoose_system_metrics_SUITE to dynamic_domains

### PR DESCRIPTION
**Pull request by vkatsuba**

### CODE REVIEW

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/73573) / small_tests / 98c368eddc912ef14aca5ec73519fe3c88faf047
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3281/73573/small_tests.24.0.5) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3281/73573/small_tests.24.0.5/small/ct_run.mongooseim@localhost.2021-09-16_17.53.15/lib.mongooseim.logs/run.2021-09-16_17.53.15/suite.log.html)

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/74352) / small_tests / b8354641e48508c1cea4fd5503eb5c78bb036914
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3281/74352/small_tests.24.0.5) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3281/74352/small_tests.24.0.5/small/ct_run.mongooseim@localhost.2021-09-17_16.06.18/lib.mongooseim.logs/run.2021-09-17_16.06.18/suite.log.html)

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/74530) / small_tests / f6a5dce4ce447b72e0c0846ba6d66815a739beba
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3281/74530/small_tests.24.0.5) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3281/74530/small_tests.24.0.5/small/ct_run.mongooseim@localhost.2021-09-20_10.11.44/lib.mongooseim.logs/run.2021-09-20_10.11.44/suite.log.html)

File: big_tests/tests/component_helper.erl [hunk 1]
```diff
@@ -148,7 +148,7 @@ mim2_service_port() ->
 
 common(_Config, Port) ->
     [{server, ct:get_config({hosts, mim, domain})},
-     {host, ct:get_config({hosts, mim, domain})},
+     {host, <<"localhost">>},
```
Comment (vkatsuba) [inline big_tests/tests/component_helper.erl:151]: As I understand external components is not supported.

File: big_tests/dynamic_domains.spec [hunk 2]
```diff
@@ -9,6 +9,11 @@
 %% http://www.erlang.org/doc/apps/common_test/run_test_chapter.html#test_specifications
 {include, "tests"}.
 
+{suites, "tests", service_mongoose_system_metrics_SUITE}.
+{skip_cases, "tests", service_mongoose_system_metrics_SUITE, 
+ [xmpp_components_are_reported], 
+ "at the moment external components doesn't support dynamic domains"}.
+
```
Comment (chrzaszcz) [inline big_tests/dynamic_domains.spec:16]: Please maintain alphabetical order of the test suites.

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/75168) / small_tests / e161ba536a6ca815c6008ff3db224c9b959e4326
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3281/75168/small_tests.24.0.5) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3281/75168/small_tests.24.0.5/small/ct_run.mongooseim@localhost.2021-09-23_06.05.27/lib.mongooseim.logs/run.2021-09-23_06.05.27/suite.log.html)

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/75194) / small_tests / 6823c8f86e943c7693fde758edfe087b7bd447e5
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3281/75194/small_tests.24.0.5) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3281/75194/small_tests.24.0.5/small/ct_run.mongooseim@localhost.2021-09-23_06.46.48/lib.mongooseim.logs/run.2021-09-23_06.46.49/suite.log.html)

**Review by Premwoik — APPROVED** [review]
Looks good to me :)

**Pull request merged**